src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -485,6 +550,97 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
+                        READ(OldFanVO(Num3)%pressureRise_str, '(F15.5)') pressureRise
+                        READ(OldFanVO(Num3)%fanTotalEff_str, '(F15.5)') fanTotalEff
+                        WRITE(OutArgs(11), '(F15.5)') (maxAirFlow * pressureRise / fanTotalEff) !- Design Electric Power Consumption {W}
+                      ELSE
+                        OutArgs(11) = 'autosize'
+                        OutArgs(12) = 'TotalEfficiencyAndPressure' ! chose this becuase power per flow or per pressure are unknown
+                      ENDIF
+                      OutArgs(13) = ''                       !- Electric Power Per Unit Flow Rate {W/(m3/s)}
+                      OutArgs(14) = ''                       !- Electric Power Per Unit Flow Rate Per Unit Pressure {W/((m3/s)-Pa)}
+                      OutArgs(15) = OldFanVO(Num3)%fanTotalEff_str          !- Fan Total Efficiency
+                      OutArgs(16) = TRIM(sysFanName) // '_curve'    !- Electric Power Function of Flow Fraction Curve Name
+                      OutArgs(17) = ''
+                      OutArgs(18) = ''
+                      OutArgs(19) = ''
+                      OutArgs(20) = ''
+                      OutArgs(21) = OldFanVO(Num3)%endUseSubCat !- End-Use Subcategory
+                      CurArgs = 21                           !- Design Electric Power Consumption {W}
+                      CALL WriteOutIDFLines(DifLfn, ObjectName, CurArgs, OutArgs, NwFldNames, NwFldUnits)
+
+                      ! create curve object
+                      ObjectName = 'Curve:Quartic'
+                      CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                      OutArgs(1) = TRIM(sysFanName) // '_curve'
+                      OutArgs(2) = OldFanVO(Num3)%coeff1             !- Coefficient1 Constant
+                      OutArgs(3) = OldFanVO(Num3)%coeff2             !- Coefficient2 x
+                      OutArgs(4) = OldFanVO(Num3)%coeff3             !- Coefficient3 x**2
+                      OutArgs(5) = OldFanVO(Num3)%coeff4             !- Coefficient4 x**3
+                      OutArgs(6) = OldFanVO(Num3)%coeff5             !- Coefficient5 x**4
+                      READ(OldFanVO(Num3)%fanPowerMinAirFlow_str, '(F15.5)') fanPowerMinAirFlow
+                      WRITE(OutArgs(7), '(F10.7)') (fanPowerMinAirFlow / maxAirFlow)  !- Minimum Value of x
+                      OutArgs(8) = '1.0'              !- Maximum Value of x
+                      OutArgs(9) = '0.0'              !- Minimum Curve Output
+                      OutArgs(10) = '5.0'             !- Maximum Curve Output
+                      OutArgs(11) = 'Dimensionless'   !- Input Unit Type for X
+                      OutArgs(12) = 'Dimensionless'   !- Output Unit Type
+                      CurArgs = 12
+                    ENDIF
+                  ENDDO
+                ENDIF
+
```

Comment:
Should the VAV fan comments be removed from in the idd and maybe the docs? Also, should the old fan be deleted during the transition?

    ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
     A7 ,  \field Supply Air Fan Object Type
        \type choice
        \key Fan:SystemModel
        \key Fan:OnOff
        \key Fan:ConstantVolume
        \key Fan:VariableVolume
        \default Fan:ConstantVolume
        \note Supply Air Fan Object Type must be Fan:SystemModel, Fan:OnOff, or Fan:ConstantVolume
        \note if AirConditioner:VariableRefrigerantFlow is used to model VRF outdoor unit
        \note Supply Air Fan Object Type must be Fan:SystemModel or Fan:VariableVolume if
        \note AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl or
        \note AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl:HR
        \note is used to model VRF outdoor unit

Reply:
> Should the VAV fan comments be removed from the idd and maybe the docs? Also, should the old fan be deleted during the transition?
Yes and Yes.